src/main/scala/org/constellation/domain/healthcheck/HealthCheckConsensusManager.scala

```diff
@@ -688,22 +690,30 @@ class HealthCheckConsensusManager[F[_]](
       else F.unit
       // mark offline those peers that we do have in our list
       toMarkOfflinePeers = peers.filterKeys(toRemove.keySet.contains).values.toList
-      _ <- peerHealthCheck.markOffline(toMarkOfflinePeers)
+      _ <- F.start(
+        peerHealthCheck.markOffline(toMarkOfflinePeers)
+      )
       // should I check the status??? I think yes as it may be that it's offline but stuck so not removed from the list and we will not add because of that
       peersToAdd = toRemain -- notOfflinePeers
       _ <- if (peersToAdd.nonEmpty)
         logger.warn(s"Peers that we don't have online but we should: ${logIds(peersToAdd.keySet)}")
       else F.unit
-      _ <- peersToAdd.toList.traverse { case (id, (_, consensus)) => addMissingPeer(id, consensus) }
+      _ <- F.start(
+        peersToAdd.toList.traverse { case (id, (_, consensus)) => addMissingPeer(id, consensus) }
```

Comment:
can `addMissingPeer` throw? if so, the whole traverse will fail. additionally, shouldn't it be
```
traverse ( F.start ... )
```
?

Reply:
All three methods: addMissingPeer, markOfflinePeer, notifyPeerAboutMissedConsensus have the error handling as the last step of the flow, so they shouldn't throw and hence interrupt the traversals I think.